tests/unit/oidc/models/test_github.py

```diff
@@ -51,6 +51,7 @@ def test_github_publisher_all_known_claims(self):
             # required unverifiable claims
             "ref",
             "sha",
+            "jti",
```

Comment:
I _think_ this should be in `__preverified_claims__` and not marked as a "required unverifiable claim."

Reply:
Moved there.